server/interactions/interaction.js

```diff
@@ -0,0 +1,27 @@
+const routes = require("express").Router();
+const axios = require("axios");
+const token = require("../../config");
+const url = "https://app-hrsei-api.herokuapp.com/api/fec2/hr-atx";
+
+routes.post("/", (req, res) => {
+  axios({
```

Comment:
primetime for a destructure here: 

```node.js
const { element, widget, time } = req.body
```

then you can just pass into your data:

```node.js
data: { element, widget, time }
```